pdp/templates/cascade.html

```diff
@@ -51,7 +51,7 @@ <h3 class="profile-label"> Official Name
                     </li>
                     <li class="list-group-item">
                         <h3 class="profile-label">UW Email</h3>
-                        <p class="userinfo" ng-repeat="email in profile.data.emails track by $index">{{ email }}</p>
+                        <p class="userinfo" ng-show="profile.data.emails.length" ng-repeat="email in profile.data.emails track by $index">{{ email }}</p>
```

Comment:
This doesn't change anything. Suggest putting the ng-show on the <li class="list-group-item> above to hide the entire element.